README.md

```diff
@@ -16,7 +16,7 @@ remove deployment platforms, or test with a different suite.
 ## Features
 * Python-centric skeletal structure with initial module files
 * Pre-configured `setup.py` for installation and packaging
-* Pre-configured Window, Linux, and OSX continuous integration on AppVeyor and Travis-CI
+* Pre-configured Window, Linux, and OSX continuous integration on AppVeyor+Travis-CI and/or GitHub Actions.
```

Comment:
I am up for removing AppVeyor and Travis entirely. @janash @Lnaden any comments?

Reply:
I am all for dropping Travis and VERY much for dropping AppVeyor. Go for it!

Reply:
Shall I remove them as part of this PR or should this be in a separate PR (also merged to `dev`) for extra clarity?